ios/Classes/ApptentiveFlutterPlugin.m

```diff
@@ -104,6 +108,7 @@ - (BOOL)apptentive_isBoolean {
 @interface ApptentiveFlutterPlugin ()
 
 @property (strong, nonatomic) FlutterMethodChannel* channel;
+@property NSData* device_token;
```

Comment:
It's probably good to add the `(strong, nonatomic)` here. `strong` is the default, but `nonatomic` is not and atomic is very slightly slower so people tend to avoid it unless it's needed.